src/Legacy/Framework/Initializer.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/**
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2015 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+
+namespace Contao\CoreBundle\Legacy\Framework;
```

Comment:
To me and @aschempp, yes :) But we'll see at the call tomorrow, thx for the changes! :)